.circleci/config.yml

```diff
@@ -280,6 +283,7 @@ jobs:
       CONBENCH_URL: "https://velox-conbench.voltrondata.run/"
       CONBENCH_HOST: "CircleCI-runner"
       LINUX_DISTRO: "centos"
+      VELOX_DEPENDENCY_SOURCE: SYSTEM
```

Comment:
I have set some of the builds to use system deps to lower job time. @kgpai let me know if you want to change these